_content/papers/beyond-reasonable-doubt_vincent-eltschinger.md

```diff
@@ -0,0 +1,19 @@
+---
+title: "Beyond Reasonable Doubt? A Note on Dharmakīrti
+and Scepticism"
+authors:
+  - "Vincent Eltschinger"
+external_url: "https://philpapers.org/archive/ELTBRD.pdf"
+drive_links:
+  - "https://drive.google.com/file/d/1GrO0-Iiz75FIchradGLURiEpuCr7Ze2I/view?usp=sharing"
+course: epistemology
+tags:
+  - scepticism
+  - stoicism
+  - logic
```

Comment:
logic is redundant with epistemology here...  Is Dharmakirti worth a whole tag unto himself? 😅